swap_meet/clothing.py

```diff
@@ -1,2 +1,14 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, condition=0, fabric="Unknown"):
+        super().__init__(id, condition)
+
+        self.fabric = fabric
+        self.category = "Clothing"
+
+    def __str__(self):
+        string1 = super().__str__()
+        string2 = f"It is made from {self.fabric} fabric."
+
+        return " ".join([string1,string2])
```

Comment:
Using your `super` method get access to the parent's classes `__str__` while at the same time overwriting it for the child class, good job! Nice use of the `join` method as well!